lib/prepareReturnValue.mjs

```diff
@@ -1,20 +1,26 @@
-/** @import {Formatter, LinterOptions, LinterResult, LintResult as StylelintResult} from 'stylelint' */
-/** @typedef {LinterOptions["maxWarnings"]} maxWarnings */
+/** @import { Formatter, LinterOptions, LinterResult, LintResult } from 'stylelint' */
 
 import emitDeprecationWarning from './utils/emitDeprecationWarning.mjs';
 
 /**
- * @param {StylelintResult[]} stylelintResults
- * @param {maxWarnings} maxWarnings
- * @param {Formatter} formatter
- * @param {string} cwd
- *
+ * @param {object} args
+ * @param {LintResult[]} args.results
+ * @param {LinterOptions['maxWarnings']} args.maxWarnings
+ * @param {LinterOptions['quietDeprecationWarnings']} args.quietDeprecationWarnings
+ * @param {Formatter} args.formatter
+ * @param {string} args.cwd
```

Comment:
The suggestion using `Required<>` does not work because of the type error below:

```
lib/standalone.mjs:124:5 - error TS2322: Type 'number | undefined' is not assignable to type 'number'.
  Type 'undefined' is not assignable to type 'number'.

124     maxWarnings,
        ~~~~~~~~~~~
```

Reply:
Then keep ` * @param {LinterOptions['maxWarnings']} args.maxWarnings` but change the rest?

Reply:
I think using `Required<>` for some and not using it for others would confuse. So, I want to keep it as-is.